src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -1,84 +1,64 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useOktaAuth } from '@okta/okta-react/dist/OktaContext';
-import { Header } from '../../common';
-import { Row, Col, InputNumber, Button, notification } from 'antd';
+import { Button } from 'antd';
 import { QuestionCircleOutlined } from '@ant-design/icons';
 import { connect } from 'react-redux';
 import { submitPoints } from '../../../api/index';
 
-import { SubmissionViewerModal } from '../../common';
-import { InstructionsModal } from '../../common';
+import { Header, SubmissionViewerModal, InstructionsModal } from '../../common';
 import { modalInstructions } from '../../../utils/helpers';
 
+import ChildRow from './ChildRow';
+import usePointShare from './usePointShare';
+
 const PointShare = props => {
   const { push } = useHistory();
-  const [totalPoints, setTotalPoints] = useState(100);
-  const [storyOnePoints, setStoryOnePoints] = useState(0);
-  const [storyTwoPoints, setStoryTwoPoints] = useState(0);
-  const [illustrationOnePoints, setIllustrationOnePoints] = useState(0);
-  const [illustrationTwoPoints, setIllustrationTwoPoints] = useState(0);
-  const [teamPoints, setTeamPoints] = useState(null);
+
+  const [pointsLeft, portfolioPoints, handleUpdatePoints] = usePointShare();
+  // TODO could probably abstract this into a useModal hook
   const [modalContent, setModalContent] = useState(null);
   const [showModal, setShowModal] = useState(false);
   const [modalVisible, setModalVisible] = useState(true);
 
+  // TODO The previous component used a useEffect that would send the submit API call if authState changed, I'm unsure if it's needed -- I haven't re-implemented it yet
   const { authState } = useOktaAuth();
 
-  const formSubmit = () => {
-    // note: lines 30 - 35 not required anymore! However, "notification" could be used for future implementations
-    // regarding error handling for the user's share points submission. Yay!
-    if (totalPoints < 0) {
-      notification.error({
-        message: 'You may only allocate 100 points!',
-      });
-      return;
-    }
-    setTeamPoints([
-      {
-        WritingPoints: storyOnePoints,
-        DrawingPoints: illustrationOnePoints,
-        MemberID: props.child.memberId,
-        SubmissionID: props.team.child1.SubmissionID,
-      },
-      {
-        WritingPoints: storyTwoPoints,
-        DrawingPoints: illustrationTwoPoints,
-        MemberID: props.child.memberId,
-        SubmissionID: props.team.child2.SubmissionID,
-      },
-    ]);
-  };
-
-  useEffect(() => {
```

Comment:
This is what you're talking about in the TODO comment, right? You should probably re-implement this as it looks like it's waiting for teamPoints to change, then submits the points with authorization. I haven't looked at the function but it seems like that's what's going on.

Reply:
Yes, but teamPoints only changes when a user pressed submit, so I figured it should just run this function when a user submits.

Reply:
Also, I'm wondering if this would mean it calls submit on initial render when the initial team state is set to null, since this useEffect is set to run when teamState changes

Reply:
Spoke over zoom. All good